coalib/processes/Processing.py

```diff
@@ -513,17 +514,30 @@ def yield_ignore_ranges(file_dict):
                                    len(file[line_number-1])))
 
                 else:
-                    for ignore_stmt in ['ignore ', 'noqa ', 'noqa']:
+                    for ignore_stmt in ['ignore\n', 'noqa\n',
```

Comment:
aren't the latter cases contained in these with `\n`?

Reply:
@Makman2